src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -1029,21 +1054,17 @@ mobile_apis::AudioStreamingState::eType StateControllerImpl::CalcAudioState(
     const mobile_apis::HMILevel::eType hmi_level) const {
   namespace HMILevel = mobile_apis::HMILevel;
   namespace AudioStreamingState = mobile_apis::AudioStreamingState;
-  using helpers::Compare;
-  using helpers::EQ;
-  using helpers::ONE;
 
   AudioStreamingState::eType audio_state = AudioStreamingState::NOT_AUDIBLE;
-  if (Compare<HMILevel::eType, EQ, ONE>(
-          hmi_level, HMILevel::HMI_FULL, HMILevel::HMI_LIMITED)) {
+  if (IsStreamableHMIType(hmi_level)) {
     if (app->IsAudioApplication()) {
       audio_state = AudioStreamingState::AUDIBLE;
     }
   }
 
   LOG4CXX_DEBUG(logger_,
-                "Calculated audio state for HMI level " << hmi_level << " --> "
-                                                        << audio_state);
+                "Calculated audio state for HMI level '"
+                    << hmi_level << "' --> '" << audio_state << "'");
```

Comment:
@EKuliiev I  would change it to more convenient one: 
` "Calculated audio state for "<< hmi_level << " hmi level is " << audio_state);`